spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaAutoConfiguration.java

```diff
@@ -89,8 +94,28 @@ public JpaSagaStore sagaStore(Serializer serializer, EntityManagerProvider entit
     @Bean
     @ConditionalOnMissingBean
     @ConditionalOnBean(DataSource.class)
-    public PersistenceExceptionResolver persistenceExceptionResolver(DataSource dataSource)
-            throws SQLException {
+    public PersistenceExceptionResolver persistenceExceptionResolver(DataSource dataSource) throws SQLException {
         return new SQLErrorCodesResolver(dataSource);
     }
+
+    @Bean(name = "deadLetterQueueProviderConfigurerModule")
+    @ConditionalOnMissingBean(name = "deadLetterQueueProviderConfigurerModule")
```

Comment:
Same here, shouldn't it be conditional on type?

Reply:
Ah, this is a left over from playing around...I'll adjust the annotations accordingly.